google/cloud/bigquery/job/load.py

```diff
@@ -629,13 +670,31 @@ def autodetect(self):
         """
         return self._configuration.autodetect
 
+    @property
+    def connection_properties(self) -> List[ConnectionProperty]:
+        """See
+        :attr:`google.cloud.bigquery.job.LoadJobConfig.connection_properties`.
+
+        .. versionadded:: 3.7.0
+        """
+        return self._configuration.connection_properties
+
     @property
     def create_disposition(self):
         """See
         :attr:`google.cloud.bigquery.job.LoadJobConfig.create_disposition`.
         """
         return self._configuration.create_disposition
 
+    @property
+    def create_session(self) -> Optional[bool]:
+        """See
+        :attr:`google.cloud.bigquery.job.LoadJobConfig.create_session`.
+
+        .. versionadded:: 3.7.0
+        """
+        return self._configuration.create_session
```

Comment:
Looks like we missed a unit test for this:

```
google/cloud/bigquery/job/load.py                              373      2     58      0    99%   680, 696
```

Reply:
yeah, looking at it